test/ussd_covid19_triage.test.js

```diff
@@ -687,6 +1155,45 @@ describe("ussd_covid19_triage app", function () {
                 })
                 .run();
         });
+        it("should ask that the info is correct confirmed contact", function () {
+            return tester
+                .setup.user.state("state_tracing")
+                .setup.user.answer("confirmed_contact", true)
+                .check.interaction({
+                    state: "state_tracing",
+                    reply: [
+                        "Finally, please confirm that the information you shared is ACCURATE to " +
+                        "the best of your knowledge?",
+                        "",
+                        "Reply",
+                        "1. YES",
+                        "2. NO"
+                    ].join("\n"),
+                    char_limit: 160
+                })
+                .run();
+        });
+        it("should display the error on invalid input confirmed contact", function () {
+            return tester
+                .setup.user.state("state_tracing")
+                .setup.user.answer("confirmed_contact", true)
+                .input("A")
+                .check.interaction({
+                    state: "state_tracing",
+                    reply: [
+                        "Please use numbers from the list.",
```

Comment:
shouldn't this error be a standard text? There are some places where "`This service works best when you select numbers from the list.`" is used... should that be the case?

Reply:
There's not enough characters to be able to show the full error text, so for certain screens it was shortened. The error text is also present in the content doc